pom.xml

```diff
@@ -190,6 +190,15 @@
                                     <excludes>META-INF/</excludes>
                                     <outputDirectory>${project.build.directory}/${wildfly.directory}/openshift/templates</outputDirectory>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.jboss.windup.web</groupId>
+                                    <artifactId>windup-web-keycloak-theme</artifactId>
+                                    <version>5.5.0-SNAPSHOT</version>
```

Comment:
Should this version number be updated in line with the main release version?

Reply:
We need to Release a new version of `windup-web-keycloak-theme` and then update this particular line. These are the dependencies that must be released before merging this PR:
- https://github.com/windup/windup-keycloak-tool/pull/15
- https://github.com/windup/windup-web-keycloak-theme/pull/6